gdal/frmts/esric/esric_dataset.cpp

```diff
@@ -0,0 +1,475 @@
+//-------------------------------------------------------------------------------
+// Name: esric_dataset.cpp
+// Purpose : gdal driver for reading Esri compact cache as raseter
+//           based on public documentation available at
+//           https://github.com/Esri/raster-tiles-compactcache
+//
+// Author : Lucian Plesea
+//
+// Udate : 06 / 10 / 2020
+//
+//  Copyright 2020 Esri
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
could you just use the standard X/MIT license used elsewhere ? Makes things simpler.